include/enet.h

```diff
@@ -5358,188 +5358,6 @@ extern "C" {
 
     #ifdef _WIN32
 
-    #ifdef __MINGW32__
```

Comment:
Let's enable compat mode by having:
```c
#if defined(__MINGW32__) && defined(ENET_MINGW_COMPAT)
```

Reply:
If there are any suggestions, I'll be happy to discuss them.

Reply:
Yeah, I think that would be great!